.circleci/config.yml

```diff
@@ -306,10 +306,20 @@ jobs:
       - checkout
       - attach_workspace:
           at: ./dist
+      - run:
+          name: "install renaming utility"
+          command: |
+            apt-get install rename
       - run:
           name: "Normalize folder names"
           command: |
             mv ./dist/macos-release ./dist/macos
+      - run:
+          name: "Rename to daily for consistency"
+          command: |
+            rename 's/\d+\.\d+\.\d+/daily/' macos/*
```

Comment:
I cannot find the exact file name we are trying to replace here (also not in the bucket AFAIS). Maybe it makes sense to write a comment what gets replaced with what. 
Is this replacing e.g. `4.7.0` to `daily`? 
Also maybe it is possible to already create it with this name? If not, then this approach is valid.

Reply:
Ah, I got it now, nevermind.

Reply:
I thought about it, but we only need this change to help with rainforest tests, I prefer to have the version on it as it helps clarify what are you downloading from the channel.